cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -186,37 +163,27 @@ def test_host_info_scvmm2016():
 @pytest.mark.tier(1)
 def test_check_disk_allocation_size_scvmm():
     """
-    https://bugzilla.redhat.com/show_bug.cgi?id=1490440
-    Steps to Reproduce:
-    1.Provision VM and check it"s "Total Datastore Used Space"
-    2.go to VMM and create Vm"s Checkpoint
-    3.open VM Details check - "Total Datastore Used Space"
+    Test datastore used space is the correct value, c.f.
+        https://github.com/ManageIQ/manageiq-providers-scvmm/issues/17
+    Note, may have to edit settings on hyper-v host for checkpoint type:
+        if set to production, try setting to standard
+
+    Bugzilla:
+        1490440
 
     Polarion:
         assignee: jdupuy
         casecomponent: Infra
         caseimportance: medium
         initialEstimate: 1/2h
         title: Check disk allocation size [SCVMM]
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(2)
-def test_vm_volume_specchar1_scvmm():
-    """
-    Special Test to verify that VMs that have Volumes with no drive letter
-    assigned don"t cause systemic SCVMM provider errors.  This is a low
```

Comment:
Question:
Is this test case automated or covers in other test case?

Reply:
Nope, not yet automated, I plan to automate most of these in the near future. This could be considered to be covered by `test_provider_summary_scvmm`, but since this test case is specific to a BZ, I think it should be its own test case.